src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -93,30 +93,34 @@ private static void Run(string[] args)
             _logger.Info("Nethermind starting initialization.");
 
             AppDomain.CurrentDomain.ProcessExit += CurrentDomainOnProcessExit;
-            IFileSystem fileSystem = new FileSystem();
-            
-            PluginLoader filePluginLoader = new("plugins", fileSystem, typeof(CliquePlugin), typeof(EthashPlugin), typeof(NethDevPlugin));
-            IPluginLoader mevLoader = SinglePluginLoader<MevPlugin>.Instance;
             
-            CompositePluginLoader pluginLoader = new (filePluginLoader, mevLoader);
-            pluginLoader.Load(SimpleConsoleLogManager.Instance);
-
-            Type configurationType = typeof(IConfig);
-            IEnumerable<Type> configTypes = new TypeDiscovery().FindNethermindTypes(configurationType)
-                .Where(ct => ct.IsInterface);
-
+            GlobalDiagnosticsContext.Set("version", ClientVersion.Version);
             CommandLineApplication app = new() { Name = "Nethermind.Runner" };
             _ = app.HelpOption("-?|-h|--help");
             _ = app.VersionOption("-v|--version", () => ClientVersion.Version, () => ClientVersion.Description);
-
-            GlobalDiagnosticsContext.Set("version", ClientVersion.Version);
-
+            
             CommandOption dataDir = app.Option("-dd|--datadir <dataDir>", "data directory", CommandOptionType.SingleValue);
             CommandOption configFile = app.Option("-c|--config <configFile>", "config file path", CommandOptionType.SingleValue);
             CommandOption dbBasePath = app.Option("-d|--baseDbPath <baseDbPath>", "base db path", CommandOptionType.SingleValue);
             CommandOption logLevelOverride = app.Option("-l|--log <logLevel>", "log level", CommandOptionType.SingleValue);
             CommandOption configsDirectory = app.Option("-cd|--configsDirectory <configsDirectory>", "configs directory", CommandOptionType.SingleValue);
             CommandOption loggerConfigSource = app.Option("-lcs|--loggerConfigSource <loggerConfigSource>", "path to the NLog config file", CommandOptionType.SingleValue);
+            _ = app.Option("-pd|--pluginsDirectory <pluginsDirectory>", "plugins directory", CommandOptionType.SingleValue);
+            
+            IFileSystem fileSystem = new FileSystem();
+            
+            string pluginsDirectoryPath = LoadPluginsDirectory(args);
+            PluginLoader filePluginLoader = new(pluginsDirectoryPath, fileSystem, 
+                typeof(CliquePlugin), typeof(EthashPlugin), typeof(NethDevPlugin));
+            
+            IPluginLoader mevLoader = SinglePluginLoader<MevPlugin>.Instance;
```

Comment:
hmmmm this should not be here, I think?

Reply:
I have added the MEV plugin by mistake, again, need to add some tests that verifies if only expected plugins are added by default